x-pack/plugin/esql/compute/gen/src/main/java/org/elasticsearch/compute/gen/AggregatorFunctionSupplierImplementer.java

```diff
@@ -139,8 +140,9 @@ private MethodSpec aggregator() {
 
         if (hasWarnings) {
             builder.addStatement(
-                "var warnings = Warnings.createWarnings(driverContext.warningsMode(), "
-                    + "warningsLineNumber, warningsColumnNumber, warningsSourceText)"
+                "var warnings = $T.createWarnings(driverContext.warningsMode(), "
+                    + "warningsLineNumber, warningsColumnNumber, warningsSourceText)",
+                WARNINGS
```

Comment:
This was a bug in the PR implementing warnExceptions on aggs, which wasn't being used anywhere yet